MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -468,8 +468,8 @@ QStringList MantidUI::getAlgorithmNames() {
   QStringList sl;
   std::vector<std::string> sv;
```

Comment:
`const auto& sv = Mantid::API::AlgorithmFactory::Instance().getKeys();` and remove the next line. Also `sv` can be renamed to `algorithmKeys` or `algorithmNames` I think